conf/nginx.conf

```diff
@@ -6,7 +6,7 @@ location = /.well-known/caldav {
 }
 
 location ^~ __PATH__ {
-  alias __FINALPATH__/;
+  alias __FINALPATH__;
```

Comment:
I'd rather prefer the common (and working for sure) fix, https://github.com/YunoHost/example_ynh/pull/45/files than removing this slash.

Reply:
Yes why not, but that will also affect the `location __PATH__ {` which happens later ... so might be less trivial ... Just removing this `/` seems be more trivial (though I haven't really checked if that does work ...)

Reply:
That's the point, I'm not sure it works without consequences. Otherwise we probably wouldn't have add a rewriting rule.

Reply:
Alrighty I tested it on a test server and in fact it's not clear that it really does solve the issue ... still shows a blank page when doing path traversal.

With the `/` at the end of the location, it does show a clear 404 error which sounds better

I commited the change :wink:

Reply:
(Also changed the other `location __PATH__` in the middle to `location __PATH__/` and it seems to be okay ... Didn't do a full clean check tho)

Reply:
Meanwhile, I try to know why package_check doesn't trig an error on nextcloud

Reply:
On LXC the default file has "Welcome to nginx" as title instead of "Welcome to nginx on Debian"...

Reply:
(╯°□°）╯︵ ┻━┻

Reply:
Also, the title changed since the time we have implement this test. Of course... It would have been too easy otherwise...